authz/api/http/transport.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package http
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"strings"
+
+	kitot "github.com/go-kit/kit/tracing/opentracing"
+	kithttp "github.com/go-kit/kit/transport/http"
+	"github.com/go-zoo/bone"
+	"github.com/mainflux/mainflux"
+	"github.com/mainflux/mainflux/authz"
+	"github.com/mainflux/mainflux/pkg/errors"
+	"github.com/opentracing/opentracing-go"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const contentType = "application/json"
+
+var errUnsupportedContentType = errors.New("unsupported content type")
+
+// MakeHandler returns a HTTP handler for API endpoints.
+func MakeHandler(svc authz.Service, tracer opentracing.Tracer) http.Handler {
+	opts := []kithttp.ServerOption{
+		kithttp.ServerErrorEncoder(encodeError),
+	}
+
+	mux := bone.New()
+
+	mux.Post("/policy", kithttp.NewServer(
```

Comment:
`/policy` or `/policies`? So far we had everything in plural (`/users`, `/things`, and so on)...